surveys/sphinx-survey-2025.yml

```
@@ -178,16 +177,17 @@ questions:
   - title: "What is the main reason you use Sphinx?"
     other: true
     choices:
-      - "Python API documentation generation (docstrings)"
-      - "Narrative documentation"
-      - "Educational materials"
+      - "API documentation generation from docstrings"
```

I'd like to make sure we're not too Python specific (Sphinx is used by large non-Python projects such as Linux), so we should disambiguate when using Python jargon.
```suggestion
      - "API documentation generation from Python docstrings"
```